.hintrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I wasn't aware of webhint before, and it's not something we currently use - if you need to have configuration in the repo itself to use it yourself, would you mind gitignoring it?

CC @tkaragianes we can investigate if any webhint checks would be useful to add to our tooling stack, but from what I've seen, I get the impression most of it winds up getting covered by other things

Reply:
I had received a message in Problems section that strict type should be used to take advantage of type safety.
I decided to not use that as doing so would cause more issues.
So, addded this as a Quick Fix help in VS code.

I agree with this comment and will remove this file from the pr.